src/main/kotlin/com/demonwav/mcdev/util/Util.kt

```diff
@@ -47,11 +46,16 @@ fun Collection<*>.toArray(): Array<Any?> {
 }
 
 @Contract(pure = true)
-inline fun <T, R> Collection<T>.mapFirstNotNull(transform: (T) -> R?): R? {
+inline fun <T, R> Iterable<T>.mapFirstNotNull(transform: (T) -> R?): R? {
     forEach { transform(it)?.let { return it } }
     return null
 }
 
+@Contract(pure = true)
+inline fun <T, R> Array<T>.mapFirstNotNull(transform: (T) -> R?): R? {
+    return asIterable().mapFirstNotNull(transform)
```

Comment:
In this case it is better to just copy the same code from above. That allows Kotlin to use the `Array.forEach` method which is compiled to a simple `for` loop instead of wrapping the array in an `Iterable` and an `Iterator`.